Clinical-Genomics/scout
# Fix #594 by manual addition of genes to dynamic panel.

This PR gives the functionality to manually add genes to dynamic "HPO" gene panels, thereby fixing #594. A user can now construct a panel of her choosing, then go directly to the variants view via the HPO/dynamic panel button, without having to first view any of the default panel genes - typically OMIM in this scenario.

<img width="596" alt="Screenshot 2019-10-02 at 20 33 31" src="https://user-images.githubusercontent.com/758570/66073807-1d1f8b00-e558-11e9-809e-ea429c692f9d.png">

**How to test**:
1. add genes to gene panel manually, e.g. POT1.
1. check that a generated HPO panel can be used in filtering on the variants page. Use a narrow term that still yields a few clinical variants, e.g. HP:0000763 on the demo case.
1. check that clinical filtering works on the variants page (for default panel)
1. check that toggling the "use HPO panel in clinical filter" sets "clinical filter" to use the HPO panel for clinical filter, via checking "HPO" in the gene list dropdown on the filter page or pressing the corresponding clinical filter button. **Edit**: *After a bit of consideration I reverted from changing the behaviour of the Clinical HPO SNV panel direct link. Its fine as it is*.

**Expected outcome**:
The functionality should be working.

**Review:**
